From 3a38985431772a021e9f160f1e9f1277f8e2d10d Mon Sep 17 00:00:00 2001
From: Anya Lin <hukk1996@gmail.com>
Date: Mon, 8 May 2023 14:51:08 +0100
Subject: [PATCH] add custom script

---
 natter-hook.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/natter-hook.sh b/natter-hook.sh
index b68227e..5b92a97 100644
--- a/natter-hook.sh
+++ b/natter-hook.sh
@@ -6,4 +6,4 @@ outter_port=$5
 
 echo "[Script] - Upload to server: ${protocol}: ${inner_ip}:${inner_port} -> ${outter_ip}:${outter_port}"
 
-# Write your upload script below...
+/etc/natter/custom-script.sh "$protocol" "$inner_ip" "$inner_port" "$outter_ip" "$outter_port"

